### ISSUE TITLE
Inventory Keep

### ISSUE THREAD

**Issue by TheOtakuJames** [graywolf336/Jail#91]
Hello,

I have multiple worlds and have it set so when a player transfers worlds, their inventory stays in the one world, not come with them. However, when I jail someone, their inventory goes with them to the normal world. Is there a way to stop this from happening?

Thanks!

-James

**Comment by graywolf336** [comment]
Have you tried to disable the inventory pieces?

**Comment by graywolf336** [comment]
@TheOtakuJames I will close this after a few days unless you respond with more details and with the answer to what have you tried so far and if you've tried to disable the inventory pieces.

**Comment by TheOtakuJames** [comment]
Ah sorry, no email came through for your first response. I have tried disabling inventory pieces, but players still have items transfer over worlds.

**Comment by graywolf336** [comment]
So, are you talking about transferring from the Jail to outside? Because when no inventory handling is taking effect then it shouldn't handle it at all..

**Comment by TheOtakuJames** [comment]
Yes, basically. Let me rephrase this using my exact problem. I have two worlds (SAO and World). When I jail someone from SAO, they go to the jail in the main world. When they leave, they are teleported back to the SAO world, but their items are no longer there.

**Comment by graywolf336** [comment]
Okay that makes sense, thanks for clearing it up. Do you use any sort of
multi-world inventory plugin?

**Comment by TheOtakuJames** [comment]
Yes, I use SeperateWorldItems.
